src/tools/wix/Pdb.cs

```diff
@@ -84,17 +84,16 @@ public static Pdb Load(string path, bool suppressVersionCheck, bool suppressSche
         internal static Pdb Load(Stream stream, Uri uri, bool suppressVersionCheck, bool suppressSchema)
         {
             XmlReader reader = null;
+            TempFileCollection tempFileCollection = null;
```

Comment:
Not critical, but I would recommend a  try/finally wherein you dispose of the TempFileCollection if it is not assigned to pdb.Output.TempFiles